src/pybind/ceph_volume_client.py

```diff
@@ -476,7 +476,7 @@ def _connect(self, premount_evict):
             self.evict(premount_evict)
             log.debug("Premount eviction of {0} completes".format(premount_evict))
         log.debug("CephFS mounting...")
-        self.fs.mount(filesystem_name=self.fs_name)
+        self.fs.mount(filesystem_name=self.fs_name.encode('utf-8'))
```

Comment:
I'm wondering if it makes more sense to move this into the moutn method: have it automatically encode its input argument from a string to a bytes array.

Reply:
I have been thinking about it. I find myself a bit confused. We can make mount method accept strings but would that make the interface inconsistent? I mean it would be a bit odd if only fs_name is expected as strings and everything else throughout is expected as bytes.

Reply:
Can you use to_bytes() function instead? And change the to_bytes() to encode with 'utf-8' explicitly.
https://github.com/ceph/ceph/pull/28557/files#diff-8625369b924524f064e083e735bd34beL25

I'm not sure if we want to change the cephfs.pyx's interfaces in this PR.  There might be community users using this library interface.

Reply:
Made the change. Would it be nice to have a `to_bytes()` in mgr/subvolumes.py as well?